ptbcontrib/ptb_jobstores/README.md

```diff
@@ -35,11 +35,23 @@ use `job_kwargs` parameter that accepts arbitrary arguments as a dictionary and
 For more information please have a look at APS scheduler's documentation about adding jobs by clicking [here](https://apscheduler.readthedocs.io/en/stable/userguide.html#adding-jobs).
 
 ## Requirements
+Main requirement includes:
 
 *   `python-telegram-bot[job-queue]~=20.0`
+
+
+For each Python ORM, you need to install the corresponding package:
+
+If you are using PyMongo:
```

Comment:
Please explain that this can be installed via `pip install ptbcontrib[ptb_jobstores_mongodb] @ …` (see the readme for the full command)

Reply:
Shouldn't it be <code>[ptb_jobstores.mongodb]</code> or something else? I don't know exactly how packaging works, but because there is no <code>ptb_jobstores_mongodb</code> package, I'm a bit confused with this naming.

I'll commit this, but if it is wrong please tell me to fix it again.